sql/src/main/java/io/crate/analyze/expressions/ExpressionAnalyzer.java

```diff
@@ -916,6 +919,49 @@ public Symbol visitObjectLiteral(ObjectLiteral node, ExpressionAnalysisContext c
                 context);
         }
 
+        @Override
+        public Symbol visitIntervalLiteral(IntervalLiteral node, ExpressionAnalysisContext context) {
```

Comment:
Is this is a good place to convert IntervalLiteral to MonthDaySecondInterval or would it better to do it in io.crate.types.IntervalType#value ?

Reply:
Calling the right constructor directly instead of going through `IntervalType.value` makes more sense to me. :+1: